index.js

```diff
@@ -35,8 +35,12 @@ function factory(prng) {
     return str
   }
 
-  function ulid() {
-    return encodeTime(Date.now(), TIME_LEN) + encodeRandom(RANDOM_LEN)
+  function ulid(seedTime) {
+    if(!seedTime) {
```

Comment:
I'd be more comfortable if this was checking for the `isNaN` condition as well, to ensure that ulid generation fails when a non-number has been passed in. For the failure condition, I'm leaning towards raising an `Error` -> what do you think?